give.php

```diff
@@ -58,6 +58,7 @@
 use Give\Form\Templates;
 use Give\FormBuilder\ServiceProvider as FormBuilderServiceProvider;
 use Give\FormMigration\ServiceProvider as FormMigrationServiceProvider;
+use Give\NotificationsHandler\ServiceProvider as NotificationHandlerServiceProvider;
```

Comment:
Looks like we are giving `NotificationsHandler` it's own top level domain in the src folder.  I'm wondering if this should actually be in the `FormBuilder` domain or if we do want to make this more general like `Notifications` to leave room for other notification type apis.

Reply:
Good point. For me, this is not entirely the Form Builder thing, but I guess it would make sense to move this to the FB domain.

Reply:
On another thought, I think having Notifications TLD might be a better option.